cli/dcoscli/log.py

```diff
@@ -116,6 +116,8 @@ def _output(curr_header, output_header, header, lines):
     if lines:
         if output_header and header != curr_header:
             emitter.publish('===> {} <==='.format(header))
+        if lines == ['']:
+            emitter.publish(DCOSException('(No logs for this task)'))
```

Comment:
We can use a `DefaultError` here instead of `DCOSException`. Lets remove the parens.